src/components/Form/PropertyForm.js

```diff
@@ -0,0 +1,347 @@
+import React, { useState,useEffect } from 'react'; 
+
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+
+import { useRouter } from 'next/navigation'
+import { useCreatePropertiesMutation,useUpdatePropertiesMutation } from '@/redux/services/propertiesApi';
+import MapComponent from '../Maps/MapComponent';
+import Button from './Button';
+import validationSchema from './SchemaValidation';
+import { Flip, ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { FormInputLabel } from '../common/FormInputLabel';
+
+const categories = [
+  { id: '1', name: 'Urbanizaciones' },
+  { id: '2', name: 'Departamentos' },
+];
+const departments = [
+  'Pando', 'Beni', 'Cochabamba', 'La Paz', 'Oruro', 'Potosí', 'Santa Cruz', 'Tarija','Sucre'
+];
+
+
+const PropertyForm = ({action,data}) => {
+  const router = useRouter()
+  const [coordinates, setCoordinates] = useState({ latitude: 0, longitude: 0 });
+  const [createProperty, { isLoading }] = useCreatePropertiesMutation();
+  const [updateProperty, { isLoading2 }] = useUpdatePropertiesMutation();
+
+
+
+  if (data){
+
+    data = {
+      ...data,
+      category: data.type
+    };
+    delete data.type;
+  }else{
+    data={
+      name: '',
```

Comment:
maybe you could create a scheme called propertyScheme and define the initial values there